src/test/datascience/dsTestSetup.ts

```diff
@@ -8,70 +8,19 @@ import { EXTENSION_ROOT_DIR_FOR_TESTS } from '../constants';
 
 const settingsFile = path.join(EXTENSION_ROOT_DIR_FOR_TESTS, 'src/test/datascience/.vscode/settings.json');
 
-function updatePackageJsonForTests() {
-    /**
-     * Modify package.json to ensure we can change logging & opt out of experiments if required, etc.
-     */
-    const packageJsonFile = path.join(EXTENSION_ROOT_DIR_FOR_TESTS, 'package.json');
-    const content = JSON.parse(fs.readFileSync(packageJsonFile).toString());
-
-    // Update package.json to pick experiments from our custom settings.json file.
-    content.contributes.configuration.properties['jupyter.experiments.optInto'].scope = 'resource';
-    content.contributes.configuration.properties['jupyter.logging.level'].scope = 'resource';
-
-    fs.writeFileSync(packageJsonFile, JSON.stringify(content, undefined, 4));
-}
-
-function updateTestsForNativeNotebooks() {
-    /**
-     * Modify package.json to ensure VSC Notebooks have been setup so tests can run.
-     * This is required because we modify package.json during runtime, hence we need to do the same thing for tests.
-     */
-    const packageJsonFile = path.join(EXTENSION_ROOT_DIR_FOR_TESTS, 'package.json');
-    const content = JSON.parse(fs.readFileSync(packageJsonFile).toString());
-
-    // This code is temporary.
-    try {
-        delete content.contributes.notebookProvider[0].priority;
```

Comment:
We don't need this either, as we don't contribute the viewer and we don't have custom/webviews anymore.